frontend/package-lock.json

```diff
@@ -1,16952 +1,8 @@
 {
```

Comment:
Thanks for reaching out about the npm version on slack! Once we get that fixed, the lockfileVersion on line 4 will hopefully go back to 2

Reply:
I just got my npm version to go up to 8.1.2" but now my package-lock.json file says this on line 4: "lockfileVersion": 1

Reply:
I forgot to mention you had to run `npm install` afterwards. I changed the version back this time, and in the future you should be good now that your npm version is up to date